### PR TITLE
qt6: 6.4.0 → 6.4.1

### PR DESCRIPTION
**Pull request by fabianhjr**
###### Description of changes

Previous nixpkgs update: #193511

Release Notes: https://code.qt.io/cgit/qt/qtreleasenotes.git/about/qt/6.4.1/release-note.md

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/qt-6/srcs.nix [hunk 1]
```diff
@@ -1,286 +1,286 @@
 # DO NOT EDIT! This file is generated automatically.
-# Command: ./maintainers/scripts/fetch-kde-qt.sh pkgs/development/libraries/qt-6
+# Command: /home/fabian/Development/Open/nixpkgs/staging-next/maintainers/scripts/fetch-kde-qt.sh pkgs/development/libraries/qt-6
```
Comment (fabianhjr) [inline pkgs/development/libraries/qt-6/srcs.nix:2]: :| wat

**Comment by fabianhjr** [comment]
Note: some packages built but not extensive. Would appreciate a nixpkgs-review since I am currently working on a gcc12 branch.

**Comment by nrdxp** [comment]
> Note: some packages built but not extensive. Would appreciate a nixpkgs-review since I am currently working on a gcc12 branch.

I'm working on one now, will be done probably in about an hour or so.

**Comment by nrdxp** [comment]
Result of `nixpkgs-review pr 202086` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.fcitx5-qt</li>
    <li>libsForQt514.fcitx5-qt</li>
    <li>obs-studio-plugins.obs-hyperion</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>obs-studio-plugins.obs-ndi</li>
  </ul>
</details>
<details>
  <summary>78 packages built:</summary>
  <ul>
    <li>beamerpresenter</li>
    <li>beamerpresenter-poppler</li>
    <li>calcmysky</li>
    <li>calibre</li>
    <li>cudaPackages.nsight_compute</li>
    <li>cutemaze</li>
    <li>duckstation</li>
    <li>fcitx5-chinese-addons</li>
    <li>fcitx5-configtool</li>
    <li>fcitx5-unikey</li>
    <li>fcitx5-with-addons</li>
    <li>jami-client-qt</li>
    <li>ladybird</li>
    <li>libsForQt5.fcitx5-qt</li>
    <li>maestral-gui</li>
    <li>mozillavpn</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>prismlauncher</li>
    <li>python310Packages.pyqt6</li>
    <li>python310Packages.pyqt6-webengine</li>
    <li>python39Packages.pyqt6</li>
    <li>python39Packages.pyqt6-webengine</li>
    <li>qMasterPassword</li>
    <li>qt6.full</li>
    <li>qt6.qmake</li>
    <li>qt6.qt3d</li>
    <li>qt6.qt5compat</li>
    <li>qt6.qtbase</li>
    <li>qt6.qtcharts</li>
    <li>qt6.qtconnectivity</li>
    <li>qt6.qtdatavis3d</li>
    <li>qt6.qtdeclarative</li>
    <li>qt6.qtdoc</li>
    <li>qt6.qthttpserver</li>
    <li>qt6.qtimageformats</li>
    <li>qt6.qtlanguageserver</li>
    <li>qt6.qtlottie</li>
    <li>qt6.qtmultimedia</li>
    <li>qt6.qtnetworkauth</li>
    <li>qt6.qtpositioning</li>
    <li>qt6.qtquick3d</li>
    <li>qt6.qtquick3dphysics</li>
    <li>qt6.qtquicktimeline</li>
    <li>qt6.qtremoteobjects</li>
    <li>qt6.qtscxml</li>
    <li>qt6.qtsensors</li>
    <li>qt6.qtserialbus</li>
    <li>qt6.qtserialport</li>
    <li>qt6.qtshadertools</li>
    <li>qt6.qtspeech</li>
    <li>qt6.qtsvg</li>
    <li>qt6.qttools</li>
    <li>qt6.qttranslations</li>
    <li>qt6.qtvirtualkeyboard</li>
    <li>qt6.qtwayland</li>
    <li>qt6.qtwebchannel</li>
    <li>qt6.qtwebengine</li>
    <li>qt6.qtwebsockets</li>
    <li>qt6.qtwebview</li>
    <li>qt6Packages.kcoreaddons</li>
    <li>qt6Packages.poppler</li>
    <li>qt6Packages.quazip</li>
    <li>qt6Packages.qxlsx</li>
    <li>qutebrowser-qt6</li>
    <li>stellarium</li>
    <li>tdesktop</li>
    <li>zint</li>
  </ul>
</details>

**Comment by nrdxp** [comment]
looks like the failure was only due to an unfree license, so we should be clear to merge.

**Pull request merged**